AI-and-Analytics/README.md

```diff
@@ -14,57 +14,7 @@ Third party program Licenses can be found here: [third-party-programs.txt](https
 
 | Type      | Folder                                             | Description
 | --------- | ------------------------------------------------ | -
-| Component | [Getting-Started-Samples](Getting-Started-Samples)               | Getting Started Samples for components in AI Kit.
-| Component & Segment | [Features-and-Functionality](Features-and-Functionality) | Demonstrate features from components like Int8 inference in Model Zoo.
+| Component | [Getting-Started-Samples](Getting-Started-Samples)               | Getting Started Samples for components in AI Tools.
+| Component & Segment | [Features-and-Functionality](Features-and-Functionality) | Demonstrate features from components like Int8 inference in Intel® Reference Models.
 | Reference | [End-to-end-Workloads](End-to-end-Workloads)                     | AI End-to-end reference workloads with real world data.
 
```

Comment:
At the end of the file we need to insert:
*Other names and brands may be claimed as the property of others. [Trademarks](http://www.intel.com/content/www/us/en/legal/trademarks.html)

This request is relevant for any README file where we use asterisks

Reply:
addressed it accordingly.